src/borg/archiver.py

```diff
@@ -1079,14 +1079,14 @@ def _delete_archives(self, args, repository):
             deleted = False
             for i, archive_name in enumerate(archive_names, 1):
                 try:
-                    del manifest.archives[archive_name]
+                    current_archive = format_archive(manifest.archives.pop(archive_name))
                 except KeyError:
                     self.exit_code = EXIT_WARNING
                     logger.warning('Archive {} not found ({}/{}).'.format(archive_name, i, len(archive_names)))
                 else:
                     deleted = True
                     msg = 'Would delete: {} ({}/{})' if dry_run else 'Deleted archive: {} ({}/{})'
-                    logger.info(msg.format(archive_name, i, len(archive_names)))
+                    logger.info(msg.format(current_archive, i, len(archive_names)))
```

Comment:
better do the format_archive() call here - also for consistency (see code below).

Reply:
okay, i'll do that and thank you for the tip with the feature branch. :)